src/act.other.cpp

```diff
@@ -2277,7 +2277,7 @@ ACMD(do_customize)
 
   skip_spaces(&argument);
   if (!*argument) {
-     send_to_char(ch, "Usage: customize <persona/physical%s%s/background>\r\n", ch, (GET_TRADITION(ch) == TRAD_SHAMANIC || GET_TRADITION(ch) == TRAD_HERMETIC) && GET_ASPECT(ch) == ASPECT_FULL ? "/reflection" : "",
+     send_to_char(ch, "Usage: customize <persona/physical%s%s/background>\r\n", ch, (GET_TRADITION(ch) == TRAD_SHAMANIC || GET_TRADITION(ch) == TRAD_HERMETIC) || (GET_ASPECT(ch) == ASPECT_FULL || GET_ASPECT(ch) == ASPECT_EARTHMAGE || GET_ASPECT(ch) == ASPECT_AIRMAGE || GET_ASPECT(ch) == ASPECT_FIREMAGE || GET_ASPECT(ch) == ASPECT_WATERMAGE) ? "/reflection" : "",
```

Comment:
Looking at this further, do you even need to have the aspect bits in here? This is basically intended to show `customize <persona/physical/reflection/background>` for people who can project, which are all shamans and hermetics AFAIK.

Reply:
That's.... a good point, I suppose it's technically the proper thing to do. There's other "flavours" of hermeticism in the books I might expand on, which have their own restrictions. So consider it hedging my bets for potential meaning down the line? Could just keep it as-was if not.

Reply:
Reviewed those other flavours, was misremembering, this can be reverted back